util/cron/common-champs.bash

```diff
@@ -32,6 +32,8 @@ module load cray-pmi
 module load cray-mpich
 module load cray-hdf5-parallel
 
+module load gcc-native/10.3
```

Comment:
Too late for this PR, but in general:  When we pin to older versions of tools, it'd be a good practice to put in a comment saying why and what we should check for before unpinning, because inevitably (often years later) we find ourselves in a situation where we're surprised we're not using the latest version and wonder why.  Having the rationale in the file to avoid having to check the git history would save some time and effort.